kafka-ui-react-app/src/components/common/table/__tests__/TableHeaderCell.spec.tsx

```diff
@@ -56,7 +56,6 @@ describe('TableHeaderCell', () => {
     const title = within(columnheader).getByRole('button');
     expect(title).toBeInTheDocument();
     expect(title).toHaveTextContent(testTitle);
-    expect(within(title).getByTitle(sortIconTitle)).toBeInTheDocument();
```

Comment:
I don't get why this line is removed

Reply:
Because now we haven't `span` with atribute `title`. Instead this I realized is like a pseudo-classes (after, before) in Title component.